tests/baselines/reference/emitExponentiationOperatorInTemplateString1.js

```diff
@@ -0,0 +1,60 @@
+//// [emitExponentiationOperatorInTemplateString1.ts]
+var t1 = 10;
+var t2 = 10;
+var s;
+`Exp: ${t1 ** t2} abc`;
```

Comment:
One important thing is that some of these tests need to vary - in a sense, it's generally more important that you test the combinations of places where a substitution expression can occur than what you actually put in them.

You should also have a test for `**`:
* At a substitution position where the template is empty: `` `${t1 ** t2}` ``
* At the first substitution position where only the *template head* is empty: `` `${t1 ** t2}hello world` ``
* At the last substitution position where the *template tail** is empty: `` `hello world${t1 ** t2 ** t2}` ``
* In two consecutive substitutions where the portions in between are empty and:
    * Head is empty: `` `${t1 ** t2}${t2 ** t1}hello world` ``
    * Tail is empty: `` `hello world${t1 ** t2}${t2 ** t1}` ``
    * Both are empty: `` `${t1 ** t2}${t2 ** t1}` ``
    * Both are full: `` `hello ${t1 ** t2}${t2 ** t1} world` ``